libcaf_net/caf/net/transport_worker_dispatcher.hpp

```diff
@@ -140,13 +143,36 @@ class transport_worker_dispatcher {
   }
 
 private:
+  worker_ptr find_by_node(const node_id& nid) {
+    if (workers_by_node_.empty())
+      return nullptr;
+    auto it = workers_by_node_.find(nid);
+    if (it == workers_by_node_.end()) {
+      CAF_LOG_ERROR("could not find worker by node: " << CAF_ARG(nid));
+      return nullptr;
+    }
+    return it->second;
+  }
+
+  worker_ptr find_by_id(const IdType& id) {
```

Comment:
```suggestion
  worker_ptr find_worker(const id_type& id) {
```

Same reasoning as above.